src/main/java/uk/gov/justice/laa/portal/landingpage/controller/UserController.java

```diff
@@ -191,15 +197,34 @@ public RedirectView setSelectedRoles(@RequestParam("selectedRoles") List<String>
 
     @GetMapping("/user/create/offices")
     public String offices(HttpSession session, Model model) {
-        OfficeData officeData = getObjectFromHttpSession(session, "officeData", OfficeData.class).orElseGet(OfficeData::new);
+        OfficeData selectedOfficeData = getObjectFromHttpSession(session, "officeData", OfficeData.class).orElseGet(OfficeData::new);
+        //if user has firms, use officeService.getOfficesByFirms();
+        List<Office> offices = officeService.getOffices();
+        List<OfficeModel> officeData = offices.stream()
```

Comment:
Is it worth using Optionals here in case some of these attributes are null in the DB?

Reply:
waiting for change of firm, for now there is no set firm. once firm set will load office based on firm then do null check